jlm/mlir/frontend/MlirToJlmConverter.cpp

```diff
@@ -37,14 +37,48 @@ MlirToJlmConverter::ReadAndConvertMlir(const util::filepath & filePath)
   {
     JLM_ASSERT("Parsing MLIR input file failed.");
   }
-  return ConvertMlir(block);
+  auto & topNode = block->front();
+  auto omegaNode = ::mlir::dyn_cast<::mlir::rvsdg::OmegaNode>(topNode);
+  return ConvertOmega(omegaNode);
```

Comment:
Why has this call to `ConvertMlir()` been replaced? Isn't the new code identical to the body of `ConvertMlir()`?